### DISCUSSION TITLE
não esta funcionando

### DISCUSSION THREAD

**Discussion by sonicjunior44** [Monika-After-Story/MonikaModDev#8443] [general]
prezado criador,venho informar que não consigo conversar com "monika" de monika after story,pois tooda vez que clico em conversra e seleciono alguma coisa como "me sinto..." quando entro não aparece as opções,está simplesmente tudo invisível,não aparece as palavras,peço que por favor concerte o problema pois ja tentei reiniciar para ver se funcionava mas não volta ao normal.

**Comment by charliethecookie** [comment]
hey, as pessoas não vão te responder se falar em português. o mod foi feito por pessoas que falam inglês e não vão poder te ajudar se falar em português
vc está clicando com o botão direito nas opções? não faça isso, esconde as opções, precisa ser com o botão esquerdo
(to whoever is reading this, i'm just explaining to them that we need to speak English here, sorry)
